README.md

```diff
@@ -10,15 +10,43 @@ The purpose of this repository is to allow easy management of all Pop!\_OS relat
 
 Binary packages are hosted [on Launchpad](https://launchpad.net/~system76/+archive/ubuntu/pop/+packages). Many packages have source on Github, under the [Pop!\_OS organization](https://github.com/pop-os).
 
+## Developer Resources
+For instructions on how to build the shell:
+	* [Cosmic, our current gnome-based shell](https://github.com/pop-os/cosmic). 
+	* [Cosmic-Epoch, the new Rust based shell](https://github.com/pop-os/cosmic-epoch).
+
 To view a list of all included repositories, see [REPOS.md](./REPOS.md)
+Developer chat: https://chat.pop-os.org/
+
+## How to Make a Change
+First of all, thank you for taking interest in contributing to our community! Every bit helps, even if it's just contributing to our documentation. 
+
+### Find the Correct Repo
+Before you make a change, you need to find the relevant repository to make a contribution in. See the 'Developer Resources' section for help finding the correct one.  When in doubt, you can make an issue in this catch all repo. 
+
+### Make An Issue
+All changes should start with an issue so the change can be properly discussed. The more complex the issue, the more discussion would be required. If an issue doesn't already exist, here would be the place to start.
+
+### Make a Pull Request.
+Fork the repository, make your changes and then make a Pull Request! Don't forget to use detail and explain what your change does. 
```

Comment:
`make your changes` should have an Oxford comma after it. "Pull request" does not need to be capitalized in the first sentence.

I'm not sure if we want the wording to be so casual as "don't forget to..." That sounds like of demeaning to me.